### PR TITLE
Add Optic

### PR DESCRIPTION
**Pull request by trulyronak**
**By submitting this pull request, I promise I have read the [contribution guidelines](https://github.com/sindresorhus/awesome-nodejs/blob/master/contributing.md) twice and ensured my submission follows it. I realize not doing so wastes the maintainers' time that they could have spent making the world better. 🖖**

⬆⬆⬆⬆⬆⬆⬆⬆⬆⬆

GitHub: https://github.com/opticdev/optic
Website: https://www.useoptic.com

Optic automatically documents changes, tests every endpoint, and gives you valuable insight into how your APIs are being used. 

Features: 

📝 Accurate API Documentation - We built Optic to make maintaining accurate specs for your APIs automatic and developer friendly. Once you add Optic to your API repo, it automatically tracks your API’s behavior and maintains a change log of its behavior over time.

It’s kind of like Git, but instead of tracking files, Optic automatically diffs and tracks API endpoints. When new behavior is observed, Optic makes it easy to update the spec.

⚙️ Automated Testing - Automate most of your contract testing. Optic uses live testing with spec coverage to make testing APIs easy.

👍 Beautiful Docs - Stripe-style documentation for every API managed by Optic.

👋 100% Open Source, Runs locally, data is only stored in your API Repo

### CODE REVIEW

**Comment by trulyronak** [comment]
@sindresorhus Could I get some feedback for why my PR hasn't been accepted? I'm happy to make any adjustments as needed :)

**Comment by sindresorhus** [comment]
> By submitting this pull request, I promise I have read the contribution guidelines twice and ensured my submission follows it.

File: readme.md [hunk 1]
```diff
@@ -338,6 +338,7 @@
 - [Docco](https://github.com/jashkenas/docco) - Documentation generator which produces an HTML document that displays your comments intermingled with your code.
 - [JSDoc](https://github.com/jsdoc3/jsdoc) - API documentation generator similar to JavaDoc or PHPDoc.
 - [Docusaurus](https://github.com/facebook/docusaurus) - Documentation website generator that leverages React and Markdown, and comes with translation and versioning features.
+- [Optic](https://github.com/opticdev/optic) - Optic automatically documents and tests your APIs
```
Comment (sindresorhus) [inline readme.md:341]: Don't include the project name in the description.

**Comment by sindresorhus** [comment]
https://github.com/sindresorhus/awesome-nodejs/commit/d08afe33de32f644d6b5e36b3eccdfdd1ef7ce29

**Pull request closed without merging**